frb_dart/lib/src/ffi/web.dart

```diff
@@ -142,3 +142,17 @@ class FrbOpaqueBase {
           OpaqueTypeFinalizer finalizer) =>
       finalizer.attach(opaque, ptr, detach: opaque);
 }
+
+int getOpaquePtr(Uint8List list) {
+  var pointBitLen = 4;
+  var ptrList = List.filled(pointBitLen, 0);
+  List.copyRange(ptrList, 0, list, 0, pointBitLen);
+  return ByteData.view(Uint8List.fromList(ptrList).buffer).getUint32(0);
+}
+
+int getOpaqueSize(Uint8List list) {
+  var pointBitLen = 4;
+  var sizeList = List.filled(pointBitLen, 0);
+  List.copyRange(sizeList, 0, list, pointBitLen, pointBitLen * 2);
+  return ByteData.view(Uint8List.fromList(sizeList).buffer).getUint32(0);
+}
```

Comment:
nit: would be great to merge the io and web implementation. (the only difference seems to be pointBitLen)